### ISSUE TITLE
The Digital Mammography DREAM Challenge

### ISSUE THREAD

**Issue by lukic-aleksandar** [ftn-ai-lab/nm-2016#3]
# Description
[The Digital Mammography DREAM Challenge](https://www.synapse.org/#!Synapse:syn4224222/wiki/401743)  will attempt to improve the predictive accuracy of digital mammography for the early detection of breast cancer. The primary benefit of this Challenge will be to establish new quantitative tools - machine learning, deep learning or other - that can help decrease the recall rate of screening mammography, with a potential impact on shifting the balance of routine breast cancer screening towards more benefit and less harm. Participating teams will be asked to submit predictive models based on over 640,000 de-identified digital mammography images from over 86000 patients, with corresponding clinical variables.
# Tech
Project will be deployed as a **Docker** image, as it is required by the challenge. 
Language: **Python**
Libs: **TensorFlow** or **Keras**
Model: Convolutional Neural Networks
# Team
**Aleksandar Lukić (E2 97/2016)**
Opened for new members.

**Comment by jocicmarko** [comment]
Odobrena tema.

Ostavi github repo link u komentaru.

**Comment by lukic-aleksandar** [comment]
Github repo: https://github.com/lukic-aleksandar/The-Digital-Mammography-DREAM-Challenge
